### PR TITLE
docs: new intro, readme update, and FAQ improvements

### LINKED ISSUE #3647: [Proposal] Introduce FAST docs with an intent driven questionnare
A great deal of confusion seems to surround what FAST is, what all of the pieces are for, and which pieces to use to achieve a specific goal in the community as awareness grows. I propose we write a questionnaire which branches out to different sections of the docs. Perhaps something like:

"What can FAST do for me?"
 - I just want components! [link to fast-components]
 - I want to build my own components [link to fast-element]
 - I want to build my own design system [design system bootstrapping template? cli?]
 - I want to extend/contribute/make new components for existing design system [new component template? cli?]

Thoughts?

### LINKED ISSUE #3669: docs: improve introduction explanation
Based on feedback from HN commenters, we should work to improve the introductory content in both our GH readme and the docs site introduction page.

### PR DESCRIPTION
**Pull request by EisenbergEffect**
# Description

This PR attempts to improve the expalanation of FAST in the documentation and readme. It also fixes up and adds a couple new FAQ entries. A small note has been added to the fast-element docs on self-closing tags.

Closes #3669 
Closes #3647 

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,17 +51,17 @@ The `@microsoft/fast-foundation` package is a library of Web Component classes,
...
 
-### fast-components-msft
-**This package has been moved to [FluentUI](https://github.com/microsoft/fluentui/) and has been renamed to `@fluentui/web-components`**
+### `@fluentui/web-components`
+
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![npm version](https://badge.fury.io/js/%40microsoft%2Ffast-components-msft.svg)](https://badge.fury.io/js/%40microsoft%2Ffast-components-msft)
+[![npm version](https://badge.fury.io/js/%40fluentui%2Fweb-components.svg)](https://badge.fury.io/js/%40fluentui%2Fweb-components)
 
 `@fluentui/web-components` is a library of Web Components that *composes* `@microsoft/fast-foundation`. `@fluentui/web-components` makes use of different stylesheets and may include components that specifically support Microsoft's Fluent design language.
```
Comment (janechu) [inline README.md:66]: Should we include a link to the Fluent UI repository?

File: sites/website/src/docs/introduction.md [hunk 2]
```diff
@@ -5,61 +5,42 @@ sidebar_label: Introduction
...
+To understand how FAST can help you, let's take a look at the FAST tech stack from top to bottom.
 
-### fast-element
+At the top of the stack FAST provides two sets of Web Components: `@fluentui/web-components` and `@microsoft/fast-components`. Both of these libraries include a common set of components found in many websites and apps. Example components include button, menu, tree-view, tab, card, modal, and more. So, what is the difference between these two libraries? The answer is that each library implements a different *design system*. `@fluentui/web-components` implement Microsoft's Fluent Design System. They look and feel like the components found in products like Windows, Office, Teams, and Edge. If you like the Fluent Design System or are building experiences designed to fit naturally into the Microsoft ecosystem, this is the component set you'll want to use. Alternatively, `@microsoft/fast-components` provides an industry-focused design system with more customization options. If you're looking to integrate FAST components into an existing site or app, or if you need more control over the theme of the components, this is the option you'll want to start with.
```
Comment (janechu) [inline sites/website/src/docs/introduction.md:26]:
```suggestion
At the top of the stack FAST provides two sets of Web Components: `@fluentui/web-components` and `@microsoft/fast-components`. Both of these libraries include a common set of components found in many websites and apps. Example components include button, menu, tree-view, tab, card, modal, and more. So, what is the difference between these two libraries? The answer is that each library implements a different *design system*. `@fluentui/web-components` implements Microsoft's Fluent Design System. They look and feel like the components found in products like Windows, Office, Teams, and Edge. If you like the Fluent Design System or are building experiences designed to fit naturally into the Microsoft ecosystem, this is the component set you'll want to use. Alternatively, `@microsoft/fast-components` provides an industry-focused design system with more customization options. If you're looking to integrate FAST components into an existing site or app, or if you need more control over the theme of the components, this is the option you'll want to start with.
```

it's hard to see my change but i changed "implement" to "implements"

File: sites/website/src/docs/introduction.md [hunk 3]
```diff
@@ -5,61 +5,42 @@ sidebar_label: Introduction
...
...component libraries. For example, if you wanted to implement Material Design, you could do that using `@microsoft/fast-foundation`. You could also implement something like Bootstrap. Both `@fluentui/web-components` and `@microsoft/fast-components` assemble the building blocks of `@microsoft/fast-foundation` to create their component sets. What types of building blocks are we talking about though? Perhaps the most valuable feature of the foundation is that it provides base component behaviors and templates for the standard components. So, if you want to implement a tree-view in your design system, for example, you would use the foundation base component behavior and template, but combine it with your own styles. The foundation components implement the state management, accessibility, keyboard navigation, and extensibility/composition model so you don't have to write that code. Additionally, foundation provides facilities for dynamic style behaviors, CSS property management, algorithmic color, RTL, high contract, and more. You don't have to write any of that. Just assemble the building blocks and add your styles to create your own component library, expressing your own design system.
```
Comment (janechu) [inline sites/website/src/docs/introduction.md:28]:
```suggestion
What if you're not just looking for a set of components to use, but you also need to implement a custom *design system*? This is where the second level of the stack comes into play. `@microsoft/fast-foundation` provides foundational building blocks that can be assembled to create new design systems and component libraries. For example, if you wanted to implement Google's Material Design, you could do that using `@microsoft/fast-foundation`. You could also implement something like Twitter Bootstrap. Both `@fluentui/web-components` and `@microsoft/fast-components` assemble the building blocks of `@microsoft/fast-foundation` to create their component sets. What types of building blocks are we talking about though? Perhaps the most valuable feature of the foundation is that it provides base component behaviors and templates for the standard components. So, if you want to implement a tree-view in your design system, for example, you would use the foundation base component behavior and template, but combine it with your own styles. The foundation components implement the state management, accessibility, keyboard navigation, and extensibility/composition model so you don't have to write that code. Additionally, foundation provides facilities for dynamic style behaviors, CSS property management, algorithmic color, RTL, high contract, and more. You don't have to write any of that. Just assemble the building blocks and add your styles to create your own component library, expressing your own design system.
```

File: packages/web-components/fast-element/docs/guide/defining-elements.md [hunk 4]
```diff
@@ -32,6 +32,10 @@ With this in place, you can now use your `name-tag` element anywhere in HTML wit
 Web Component names must contain a `-` in order to prevent future conflicts with built-in elements and to namespace components from different libraries. For more information on the basics of Web Components [see this set of articles](https://developers.google.com/web/fundamentals/web-components).
 :::
 
+:::note
+HTML has a few special tags known as "self-closing tags." Common examples include `<input>` and `<img>`. However, most HTML elements and **all** web components must have an explicit closing tag.
```
Comment (janechu) [inline packages/web-components/fast-element/docs/guide/defining-elements.md:36]:
```suggestion
HTML has a few special tags known as "self-closing tags". Common examples include `<input>` and `<img>`. However, most HTML elements and **all** web components must have an explicit closing tag.
```

File: README.md [hunk 5]
```diff
@@ -25,32 +31,18 @@ This is the FAST monorepo, containing web component packages, tools, examples, a
...
-
-Let's take a look at what each of FAST's core packages gives us today.
-
-### fast-element
+### `@microsoft/fast-element`
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Ffast-element.svg)](https://badge.fury.io/js/%40microsoft%2Ffast-element)
 
 The `@microsoft/fast-element` library is a lightweight means to easily building performant, memory-efficient, standards-compliant Web Components. FAST Elements work in every major browser and can be used in combination with any front-end framework or even without a framework. To get up and running with `@microsoft/fast-element` see [the Getting Started guide](http://fast.design/docs/fast-element/getting-started).
```
Comment (Falkicon) [inline README.md:43]: I think this should be "lightweight means to easily build performant"

File: README.md [hunk 6]
```diff
@@ -59,17 +51,17 @@ The `@microsoft/fast-foundation` package is a library of Web Component classes,
 
 This package does not export Web Components registered as [custom elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) - it exports parts and pieces intended to be *composed* into Web Components, allowing you to implement your own design language by simply applying CSS styles and behaviors without having to write all the JavaScript that's involved in building production-quality component implementations.
 
-### fast-components
+### `@microsoft/fast-components`
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Ffast-components.svg)](https://badge.fury.io/js/%40microsoft%2Ffast-components)
 
 `@microsoft/fast-components` is a library of Web Components that *composes* the exports of `@microsoft/fast-foundation` with stylesheets aligning to the FAST design language. This composition step registers a custom element. See the [quick start](http://fast.design/components/getting-started) to get started using the components.
```
Comment (Falkicon) [inline README.md:59]: Should be "aligning to the FAST Frame design system."

File: README.md [hunk 7]
```diff
@@ -17,6 +17,12 @@
 
 This is the FAST monorepo, containing web component packages, tools, examples, and documentation. FAST tech can be used à la carte or as a suite to build enterprise-grade websites, applications, components, design systems, and more.
 
+:star: We appreciate your star, it helps!
+
+## Introduction
+
+FAST is a collection of technologies built on Web Components and other modern Web Standards, designed to help you efficiently tackle some of the most common challenges in website and application design and development.
```
Comment (awentzel) [inline README.md:24]: could remove "other"

File: sites/website/src/docs/resources/faq.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src
 
 ### Who is behind FAST?
 
-The Microsoft FAST team builds and maintains all the `@microsoft/fast-*` packages. We are a collection of UX Engineers and Designers who are passionate about solving real-world UX challenges using web standard technologies. You can find us on GitHub [here](https://github.com/microsoft/fast).
+The Microsoft FAST team builds and maintains all the `@microsoft/fast-*` packages as well as the `@fluentui/web-components` package. We are a collection of UX Engineers and Designers who are passionate about solving real-world UX challenges using web standard technologies. You can find us on GitHub [here](https://github.com/microsoft/fast).
```
Comment (awentzel) [inline sites/website/src/docs/resources/faq.md:10]: you could link "GitHub" and remove "here".

**Pull request merged**